templates/datasource.sh.j2

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env bash
+
+read -d '' template << EOF
+		<datasources>
+			<datasource jndi-name="java:jboss/datasources/ExampleDS" pool-name="ExampleDS" enabled="true" use-java-context="true" statistics-enabled="false">
+                    <connection-url>jdbc:h2:mem:test;DB_CLOSE_DELAY=-1;DB_CLOSE_ON_EXIT=FALSE</connection-url>
+                    <driver>h2</driver>
+                    <security>
+                        <user-name>sa</user-name>
+                        <password>sa</password>
+                    </security>
+                </datasource>
```

Comment:
Is this required?

Reply:
apparently, it was failing to start without this config. I could check one again and research